packages/azure-themes/src/azure/AzureThemeHighContrastLight.ts

```diff
@@ -0,0 +1,138 @@
+  primaryButtonTextHovered: HighContrastLightSemanticColors.primaryButton.hover.text,
+  primaryButtonTextPressed: HighContrastLightSemanticColors.primaryButton.pressed.text,
+  primaryCompoundButtonBorder: HighContrastLightSemanticColors.primaryButton.rest.border,
+  radioButtonPillBorderDisabled: HighContrastLightSemanticColors.radioButton.circle.borderDisabled,
+  radioButtonPillCheckedHover: HighContrastLightSemanticColors.radioButton.pill.checkedHover,
+  radioButtonPillUncheckedDisabled: HighContrastLightSemanticColors.radioButton.pill.uncheckedDisabled,
+  radioButtonPillUncheckedHover: HighContrastLightSemanticColors.radioButton.pill.uncheckedHover,
+  radioButtonPillDisabled: HighContrastLightSemanticColors.radioButton.pill.disabled,
+  radioButtonThumbUnchecked: HighContrastLightSemanticColors.radioButton.circle.uncheckedRest,
+  radioButtonThumbUncheckedDisabled: HighContrastLightSemanticColors.radioButton.circle.checkedDisabled,
+  radioButtonThumbCheckedDisabled: HighContrastLightSemanticColors.radioButton.circle.checkedDisabled,
+  rowBorder: HighContrastLightSemanticColors.radioButton.circle.checkedDisabled,
+  toggleDisabledBackground: HighContrastLightSemanticColors.toggle.disabled.backrgound,
+  variantBorder: HighContrastLightSemanticColors.controlOutlines.rest,
+  // extended
+  controlAccent: HighContrastLightSemanticColors.controlOutlines.accent,
+  controlOutline: HighContrastLightSemanticColors.controlOutlines.rest,
+  controlOutlineDisabled: HighContrastLightSemanticColors.controlOutlines.disabled,
+  controlOutlineHovered: HighContrastLightSemanticColors.controlOutlines.hover,
+  iconButtonBackground: HighContrastLightSemanticColors.primaryButton.hover.background,
+  iconButtonFill: HighContrastLightSemanticColors.text.icon,
+  iconButtonFillHovered: HighContrastLightSemanticColors.primaryButton.hover.text,
+  labelText: HighContrastLightSemanticColors.text.label,
+  statusErrorBackground: HighContrastLightSemanticColors.statusBar.error,
+  statusErrorText: HighContrastLightSemanticColors.text.body,
+  statusErrorIcon: CommonSemanticColors.icons.error,
+  statusInformationBackground: HighContrastLightSemanticColors.statusBar.information,
+  statusInformationText: HighContrastLightSemanticColors.text.body,
+  statusInformationIcon: CommonSemanticColors.icons.information,
+  statusSuccessBackground: HighContrastLightSemanticColors.statusBar.okay,
+  statusSuccessText: HighContrastLightSemanticColors.text.body,
+  statusSuccessIcon: CommonSemanticColors.icons.okay,
+  statusWarningBackground: HighContrastLightSemanticColors.statusBar.warning,
+  statusWarningText: HighContrastLightSemanticColors.text.body,
+  statusWarningIcon: CommonSemanticColors.icons.warning,
+  textFieldBorderDisabled: HighContrastLightSemanticColors.text.disabled,
+  textFieldBackgroundDisabled: HighContrastLightSemanticColors.disabledButton.background,
+
+  // temporary work around for high contrast themes
```

Comment:
Same as above comment. Mention a workitem number to fix this workaround, if applicable.